docs/sources/tempo/configuration/_index.md

```diff
@@ -1054,29 +1070,11 @@ storage:
             [queue_depth: <int>]
 
         # Configuration block for the Write Ahead Log (WAL)
-        wal:
-
-            # where to store the head blocks while they are being appended to
-            # Example: "wal: /var/tempo/wal"
-            [path: <string>]
-
-            # wal encoding/compression.
-            # options: none, gzip, lz4-64k, lz4-256k, lz4-1M, lz4, snappy, zstd, s2
-            [v2_encoding: <string> | default = snappy]
-
-            # Defines the search data encoding/compression protocol.
-            # Options: none, gzip, lz4-64k, lz4-256k, lz4-1M, lz4, snappy, zstd, s2
-            [search_encoding: <string> | default = none]
-
-            # When a span is written to the WAL it adjusts the start and end times of the block it is written to.
-            # This block start and end time range is then used when choosing blocks for search.
-            # This is also used for querying traces by ID when the start and end parameters are specified. To prevent spans too far
-            # in the past or future from impacting the block start and end times we use this configuration option.
-            # This option only allows spans that occur within the configured duration to adjust the block start and
-            # end times.
-            # This can result in trace not being found if the trace falls outside the slack configuration value as the
-            # start and end times of the block will not be updated in this case.
-            [ingestion_time_range_slack: <duration> | default = 2m]
+        wal: <WAL Config>
+          [path: <string> | default = "/var/tempo/wal"]
+          [v2_encoding: <string> | default = snappy]
+          [search_encoding: <string> | default = none]
+          [ingestion_time_range_slack: <duration> | default = 2m]
```

Comment:
Needed a way to show that this particular component is overriding values from the base component block. (WAL Config)

Wanted to keep it light for the future, so no comments where none would add additional value.  
If there were comments we would need to keep both synchronized.

This way if you are making changes to the `query-frontend` you won't need to change the base block becuase it isn;t dependant on the `query-frontend`  
Same in referse, if you made changes to the base `WAL Config` you wouldn't need to propagate them to every component that use it. (Unless you made changes to them too)

Feel free to dispute the ommission of the newline breaks.  
> I felt a need to keep this visually distinct, but I am not married to the idea.  
> Might be better to keep the format consistent